File: src/main/java/ru/zdoher/japs/repositories/textbook/TextbookRepository.java
package ru.zdoher.japs.repositories.textbook;

import org.springframework.data.repository.CrudRepository;
import org.springframework.stereotype.Repository;
import ru.zdoher.japs.domain.textbook.Textbook;

import java.util.List;

@Repository
public interface TextbookRepository
        extends CrudRepository<Textbook, String> {
    List<Textbook> findAll();
}

File: src/main/java/ru/zdoher/japs/repositories/textbook/TextbookRepository.kt
package ru.zdoher.japs.repositories.textbook

import org.springframework.data.mongodb.repository.ReactiveMongoRepository
import org.springframework.stereotype.Repository
import ru.zdoher.japs.domain.textbook.Textbook

@Repository
interface TextbookRepository : ReactiveMongoRepository<Textbook, String>